components/apimgt/org.wso2.carbon.apimgt.core/src/test/java/org/wso2/carbon/apimgt/core/dao/impl/ApiDAOImplIT.java

```diff
@@ -596,10 +596,10 @@ public void testGetAPIsByStatus() throws Exception {
             apiDAO.addAPI(api);
         }
 
-        List<API> blockedAPIsSummary = new ArrayList<>();
-        for (int i = 0; i < numberOfBlocked; ++i) {
-            API api = SampleTestObjectCreator.createUniqueAPI().lifeCycleStatus(blockedStatus).build();
-            blockedAPIsSummary.add(SampleTestObjectCreator.getSummaryFromAPI(api));
+        List<API> maintenanceAPIsSummary = new ArrayList<>();
+        for (int i = 0; i < numberOfInMaintenance; ++i) {
```

Comment:
@Niranjan-K Yes currently it is 1. But the point is we can choose to change the value of numberOfInMaintenance at a later point. Variables such as numberOfInMaintenance act as parameters for the test. So by writing the code in this way we can change the parameters that goverm the test and the code will still work. When you look at it in the context of a test case, this makes sense.